iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/DanglingLineImpl.java

```diff
@@ -6,18 +6,226 @@
  */
 package com.powsybl.iidm.network.impl;
 
-import com.powsybl.iidm.network.CurrentLimitsAdder;
-import com.powsybl.iidm.network.ConnectableType;
-import com.powsybl.iidm.network.DanglingLine;
-import com.powsybl.iidm.network.ValidationUtil;
+import com.powsybl.commons.util.trove.TBooleanArrayList;
+import com.powsybl.iidm.network.*;
 import com.powsybl.iidm.network.impl.util.Ref;
 import gnu.trove.list.array.TDoubleArrayList;
 
 /**
  *
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-class DanglingLineImpl extends AbstractConnectable<DanglingLine> implements DanglingLine, CurrentLimitsOwner<Void> {
+class DanglingLineImpl extends AbstractConnectable<DanglingLine> implements DanglingLine, CurrentLimitsOwner<Void>, ReactiveLimitsOwner {
+
+    class GenerationImpl implements Generation, MultiVariantObject {
```

Comment:
I think it's not necessary to inherit from MultiVariantObject. If the enclosing object is alreay a MultiVariantObject it could be ok

Reply:
I'll change it